pinot-core/src/main/java/org/apache/pinot/core/io/writer/SingleColumnMultiValueWriter.java

```diff
@@ -19,60 +19,14 @@
 package org.apache.pinot.core.io.writer;
 
 public interface SingleColumnMultiValueWriter extends DataFileWriter {
-  /**
```

Comment:
we don't want to remove these. There are usecases where we will support no-dictionary modes

Reply:
Same here. I only removed the ones that does not apply to Pinot data types.